### ISSUE TITLE
Responsive issue in admin section's ui

### ISSUE THREAD

**Issue by skanarul8002** [skanarul8002/mca_hub#12]
Login as guest as admin and browse all section

**Comment by n-keerthi-gayathri** [comment]
@skanarul8002 please assign me this issue

**Comment by AKASH7233** [comment]
@skanarul8002 can you assign this issue to me?

**Comment by skanarul8002** [comment]
Please @AKASH7233, You must star ⭐ the repository first before contributing

**Comment by AKASH7233** [comment]
> Please @AKASH7233, You must star ⭐ the repository first before contributing

okay !

**Comment by AKASH7233** [comment]
@skanarul8002  can you specify which parts of admin section need to be fix ?

**Comment by skanarul8002** [comment]
Yeah @AKASH7233, you can choose at least one section and then work on it. Hope you understand.

**Comment by AKASH7233** [comment]
yeah !

**Comment by n-keerthi-gayathri** [comment]
@skanarul8002  i want to make them responsive

-Admin/classes

-Admin/subjects

-Admin/teachers

**Comment by n-keerthi-gayathri** [comment]
> @skanarul8002 i want to make them responsive
> 
> -Admin/classes
> 
> -Admin/subjects
> 
> -Admin/teachers
@skanarul8002 please assign me this issue
